common/os.cpp

```diff
@@ -248,6 +249,8 @@ Some auxillary functionalities might be affected. Please report this message if
     {
         std::string res;
 #ifdef _WIN32
+        using namespace librealsense::platform;
```

Comment:
For `check`? Then please name the class specifically: `using librealsense::platform::check`.
Just make sure this still compiles and links in shared mode...

Reply:
Looks like we have unresolved externals in some of the builds...
No good deed goes unpunished. /sigh
I guess replace CHECK_HR by the contents of check again? or move some variant of check() into `common/utilities/windows/hresult.h`...

Reply:
I'm not sure I understood but I fixed the unresolved externals